subprojects/docs/src/samples/testing/junitplatform/jupiter/build.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'java'
```

Comment:
This sample needs to be tested similar to the ones we have under `org.gradle.integtests.samples` in the `integ-test` module.

Reply:
Thanks. Tests for samples are now in `JUnitPlatformSampleIntegrationTest` class.